scripts/travis.sh

```diff
@@ -5,8 +5,9 @@ SUBFOLDER=$1
 
 function test {
     set -ev
-    cd $TRAVIS_BUILD_DIR/$SUBFOLDER
     npm install
+    npm run build
+    cd $TRAVIS_BUILD_DIR/$SUBFOLDER
     npm test
```

Comment:
I think there's a `npm install` missing here after the `cd`. But that will change in #626 anyway.